lib/livebook/zta/cloudflare.ex

```diff
@@ -69,11 +71,15 @@ defmodule Livebook.ZTA.Cloudflare do
   defp verify_iss(%{fields: %{"iss" => iss}}, iss), do: :ok
   defp verify_iss(_, _), do: :error
 
-  defp get_user_identity(token, fields, url) do
+  defp get_user_identity(token, url) do
     token = "CF_Authorization=#{token}"
-    fields = Enum.map(fields, &Atom.to_string/1)
     resp = Req.request!(url: url, headers: [{"cookie", token}])
-    if resp.status == 200, do: {:ok, Map.take(resp.body, fields)}, else: :error
+    if resp.status == 200, do: {:ok, Map.take(resp.body, Map.keys(@fields))}, else: :error
```

Comment:
I don't think we need to validate all fields exist. I think we can say that the fields are not guarantee to be returned and the caller can choose to raise. In this case, I would implement this code like this:

```suggestion
    if resp.status == 200, do: {:ok, for({k, v} <- resp.body, new_k = @fields[k], do: {new_k, v}, into: %{})}, else: :error
```